### ISSUE TITLE
What about gcd(0,0) value?

### ISSUE THREAD

**Issue by askryl** [yadudoc/erlang#1]
https://github.com/yadudoc/erlang/blob/54a117a242c61522ef48a71f02489d860d7e3620/l-99/gcd.erl#L6

**Comment by enok3n** [comment]
> https://github.com/yadudoc/erlang/blob/54a117a242c61522ef48a71f02489d860d7e3620/l-99/gcd.erl#L6

yes, it just generates an exception when passing 0 to either a or b. the solution works only for happy path
